pep-0696.rst

```diff
@@ -160,19 +161,21 @@ literal "``...``".
 
    DefaultTs = TypeVarTuple("DefaultTs", default=Unpack[tuple[str, int]])
 
-   class Foo(Generic[DefaultTs]): ...
+   class Foo(Generic[*DefaultTs]): ...
 
    reveal_type(Foo())             # type is Foo[str, int]
    reveal_type(Foo[int, bool]())  # type is Foo[int, bool]
 
-Using another ``TypeVarLike`` as the default
+Using Another ``TypeVarLike`` as ``default``
 ''''''''''''''''''''''''''''''''''''''''''''
 
-To use another ``TypeVarLike``\ s as the default they have to be of the
-same type. When using another ``TypeVarLike`` (T1) as the default, the default
-for the ``TypeVarLike`` (T2), T1 must be used before in the signature
-of the class it appears in before T2. T2's bound must be a subtype of
-T1's bound.
+This allows for a value to be used again when the constraints solver
+fails to solve a constraint for a type or the a type parameter to a
+generic is missed out but another is specified.
```

Comment:
```suggestion
fails to solve a constraint for a type, or the type parameter to a
generic is missing but another is specified.
```

There appears to be a typo here, and a missing comma. Also, I couldn't figure out precisely what is meant by "another" here—another _what_ is specified?

Reply:
Thanks, that reads better, I missed out type parameter so this now reads
> fails to solve a constraint for a type, or the type parameter to a
> generic is missing but a different type parameter is specified.

which I think is clearer